### PR TITLE
escape!

### PR DESCRIPTION
**Pull request by rosalindfdt**
did escape from the computer battlegames book.

### CODE REVIEW

File: docs/books/computer-games/computer-battle-games/escape/index.rst [hunk 1]
```diff
@@ -1,11 +1,26 @@
-=======
+=============
 Escape!
-=======
+=============
 
+The Robots have caught you, taken your weapons and locked you up. Suddenly you remember you still have your sonar wristwatch, which can be tuned to produre sounds of any frequency. If you can only find the resonant frequency of your Robot guards, they should vibrate so much they fall apart.
+
+You must be carefus not to use frequencies that are too low or the building will vibrate and collapse on top of you. If you go too high, you will get such a headache you will have to give up.
```
Comment (freakboy3742) [inline docs/books/computer-games/computer-battle-games/escape/index.rst]: typo: carefus

File: docs/books/computer-games/computer-battle-games/escape/index.rst [hunk 2]
```diff
@@ -1,11 +1,26 @@
-=======
+=============
 Escape!
-=======
+=============
```
Comment (freakboy3742) [inline docs/books/computer-games/computer-battle-games/escape/index.rst]: As a stylistic thing, the underlining should match the length of the line being underlined. Most ReST compilers won't complain, but some do.

File: docs/books/computer-games/computer-battle-games/escape/escape.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import os, random
+
+os.system('cls||clear')
```
Comment (freakboy3742) [inline docs/books/computer-games/computer-battle-games/escape/escape.py]: There's another way to tackle this, using ANSI sequences for the TTY - see [Beat the Bug Eyes](https://github.com/pybee/yorkshire4/blob/master/docs/books/computer-games/computer-space-games/beat_the_bug_eyes/beat_the_bug_eyes.py#L46) for an example. 

Just copy-pasting the clear method is fine for now; but ideally, utilities like `clear()`, `getch()`, etc would be factored out into a support library, and we'd have some docs on how to create a virtual environment and install the support library.

**Comment by rosalindfdt** [comment]
fixed the issues in the comments!

**Pull request merged**